### PR TITLE
py-terminado: add 0.17.1

### PR DESCRIPTION
**Pull request by manuelakuhn**
https://github.com/jupyter/terminado/tree/v0.17.1

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-terminado/package.py [hunk 1]
```diff
@@ -10,22 +10,27 @@
 class PyTerminado(PythonPackage):
     """Terminals served to term.js using Tornado websockets"""
 
+    homepagge = "https://github.com/jupyter/terminado"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-terminado/package.py]:
```suggestion
    homepage = "https://github.com/jupyter/terminado"
```

**Review by tldahlgren — COMMENTED** [review]
Confirmed the version sha256 and existence of the home page URL.

Please fix the typo.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Selecting `Request changes` for the typo fix.

**Review by adamjstewart — APPROVED** [review]
Oh right, this is the package that switched from distutils to flit to setuptools to hatchling. Excited to see what new backend they choose next!

**Review by tldahlgren — APPROVED** [review]
Thanks for the fix.

**Pull request merged**